rebar.config.erl

```diff
@@ -268,3 +268,27 @@ str(B) when is_binary(B) -> unicode:characters_to_list(B, utf8).
 
 erl_opts_i() ->
     [{i, "apps"}] ++ [{i, Dir}  || Dir <- filelib:wildcard(filename:join(["apps", "**", "include"]))].
+
+dialyzer(Config) ->
+    {dialyzer, OldDialyzerConfig} = lists:keyfind(dialyzer, 1, Config),
+
+    AppsToExclude = case os:getenv("DIALYZER_ANALYSE_APP") of
+        false ->
```

Comment:
add another clause to match `""` which is what happens when variable is set to empty value.